src/createBaseForm.js

```diff
@@ -452,7 +452,7 @@ function createBaseForm(option = {}, mixins = []) {
           const name = field.name;
           if (options.force !== true && field.dirty === false) {
             if (field.errors) {
-              set(alreadyErrors, name, { errors: field.errors });
+              alreadyErrors[name] = { errors: field.errors };
```

Comment:
Sorry for the late response, it seems that your PR will change the structure of [the first argument of `validateFields`](https://github.com/react-component/form/blob/e9e2359a34044799cf92ea522c3b4fdb6fc878e5/src/createBaseForm.js#L476)